packages/block-editor/src/components/media-replace-flow/test/index.js

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { act } from 'react-dom/test-utils';
```

Comment:
Is there any reason for not using `act` and `render` from the `@testing-library/react`?

Reply:
Me being a noob.

Reply:
I basically check this article every time I've to write unit tests 😅  - https://kentcdodds.com/blog/common-mistakes-with-react-testing-library